app/src/main/java/fr/free/nrw/commons/theme/NavigationBaseActivity.java

```diff
@@ -187,14 +189,19 @@ public boolean onNavigationItemSelected(@NonNull final MenuItem item) {
                 return true;
             case R.id.action_feedback:
                 drawerLayout.closeDrawer(navigationView);
+
+                String technicalInfo = new CommonsLogSender(sessionManager,
```

Comment:
You do not need to create a new instance of `CommonsLogSender`. 

Simply add 

```
@Inject CommonsLogSender commonsLogSender 
```

Reply:
@maskaravivek Done 👍, thanks for the suggestion :)